### PR TITLE
Setup library metadata

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: library.lp [hunk 1]
```diff
@@ -0,0 +1,10 @@
+(librepcb_library 0120d71f-d2f7-4e52-8a11-80ada981ad5f
+ (name "Adafruit")
+ (description "Adafruit breakout boards. Inofficial")
+ (keywords "")
```
Comment (dbrgn) [inline library.lp]:
```suggestion
 (keywords "adafruit")
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,10 @@
-# Adafruit.lplib
-Adafruit boards. Inofficial.
+# Adafruit
+
+## Description
+
+This is a [LibrePCB](https://librepcb.org) library!
+Just edit this file to add a description about it.
```
Comment (dbrgn) [inline README.md]: If you want, add a description here :wink:

**Review by dbrgn — DISMISSED** [review]

File: README.md (hunk 2 above)
Comment (ghost) [reply README.md]: Done.

**Review by rnestler — DISMISSED** [review]

File: library.lp [hunk 3]
```diff
@@ -0,0 +1,10 @@
+(librepcb_library 0120d71f-d2f7-4e52-8a11-80ada981ad5f
+ (name "Adafruit")
+ (description "Adafruit breakout boards. Inofficial")
```
Comment (dbrgn) [inline library.lp]: For consistency with other libs:

```suggestion
 (description "Adafruit breakout boards (inofficial).")
```

**Review by dbrgn — CHANGES_REQUESTED** [review]
One last change request :slightly_smiling_face: Then it's good to merge.

File: library.lp (hunk 3 above)
Comment (rnestler) [reply library.lp]: Hmm. I noticed it is less consistent in the repository names. But that is not really related.
Edit: I adapted the repository descriptions from Adafruit and Raspberry Pi as well.

**Review by dbrgn — APPROVED** [review]

**Pull request merged**